tests/Unit/Helpers/LocalizedRouteTest.php

```diff
@@ -27,3 +28,12 @@
     expect(localizedRoute('via.model.default', ['foo' => 'bar']))
         ->toEndWith('localhost/model/default/bar');
 });
+
+test('routes hide fallback', function () {
+    config()->set(Name::Shared() . '.routes.hide_default', true);
+    config()->set('app.locale', \LaravelLang\LocaleList\Locale::Maori->value);
+    config()->set('app.fallback_locale', \LaravelLang\LocaleList\Locale::Maori->value);
```

Comment:
This test does not make sense as it should check if the redirect is correct. In this variant it is impossible to check it. Localizations should be different.

FQN should also be used.

Reply:
French and German are used in the tests. It is better to stick to the current values.